maxim/logger/components/generation.py

```
@@ -11,16 +11,22 @@
 from .attachment import FileAttachment, FileDataAttachment, UrlAttachment
 from .base import BaseContainer
 from .types import Entity, GenerationError, GenerationErrorTypedDict, object_to_dict
+from .utils import parse_attachments_from_messages
 
 
-class GenerationRequestMessageContent(TypedDict):
-    type: str
+class GenerationRequestTextMessageContent(TypedDict):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)